system-tests/project-fixtures/svelte/src/Counter.svelte

```diff
@@ -0,0 +1,20 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+
+	const dispatch = createEventDispatcher();
```

Comment:
```suggestion
  const dispatch = createEventDispatcher();
```

Reply:
Weird looks totally fine on my screen, must have done something weird with tabs/spaces or the svelte extension hijacked it.